mount/losetup_linux.go

```diff
@@ -0,0 +1,230 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package mount
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"strings"
+	"syscall"
+	"time"
+	"unsafe"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	loopControlPath = "/dev/loop-control"
+	loopDevFormat   = "/dev/loop%d"
+
+	// According to util-linux/include/loopdev.h
+	ioctlSetFd       = 0x4C00
+	ioctlClrFd       = 0x4C01
+	ioctlSetStatus64 = 0x4C04
+	ioctlGetFree     = 0x4C82
+
+	loFlagsReadonly = 1
+	//loFlagsUseAops   = 2
+	loFlagsAutoclear = 4
+	//loFlagsPartScan  = 8
+	loFlagsDirectIO = 16
+
+	ebusyString = "device or resource busy"
```

Comment:
unix.EBUSY.Error() is this. But I think you don't need to compare the error string.